src/containers/pages/FocusInvestigation/map/active/index.tsx

```diff
@@ -331,10 +333,10 @@ class SingleActiveFIMap extends React.Component<
           <div className="col-3">
             <div className="mapSidebar">
               <div>
-                <h5>
+                <h1 className="h5">
```

Comment:
This is a somewhat odd class name for a h1 element, no?

Reply:
Agreed, this should either be `h3` or remain `h5`. If this were in an actual html5 `aside` element then that would be semantically different, but as is shouldn't be `h1`.

Reply:
I think the other option would be to use the `p` tag and add the className of h5

Reply:
just found I actually did not need to make changes to that line.